### DISCUSSION TITLE
Comparison between whisper.cpp and Apple Speech framework

### DISCUSSION THREAD

**Discussion by MikePendo** [ggml-org/whisper.cpp#2199] [Q&A]
I am looking for speech to text (ASR) for mobile devices. I have seen a lot of good reviews about whisper. 
I am wondering if any one did a comparison between Whisper and Apple Speech framework in relation to mobile?

**Comment by sameersismail** [comment]
Also interested to know this.

**Comment by sharlec** [comment]
also have interests on this comparison
